taggit/managers.py

```diff
@@ -309,13 +339,18 @@ def __lt__(self, other):
         return False
 
     def post_through_setup(self, cls):
-        self.related = RelatedObject(cls, self.model, self)
+        if RelatedObject is not None:  # Django < 1.8
```

Comment:
Interesting. If @dtiesling is running into that problem, I bet others will too. I wonder if we should use the VERSION method just to be safe.